### PR TITLE
Add Redlinks plugin

### PR DESCRIPTION
**Pull request by Mortal**
Simple implementation of #485. Checks each `<a>` tag in the tree to see if the `href` points to a wiki article that does not exist; if so, the CSS class `wiki-broken` colors the link red (#ba0000, same color as Wikipedia uses).

### CODE REVIEW

**Comment by Mortal** [comment]
Is the file `src/wiki/static/wiki/css/all.css` actually used by django-wiki anymore? It doesn't seem to be updated by `make assets`.

File: tests/plugins/redlinks/test_redlinks.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+from wiki.core import markdown
+
+
+class RedlinksTests(RequireRootArticleMixin, TestBase):
+    def test_internal(self):
+        md = markdown.ArticleMarkdown(article=self.root_article)
+        md_text = md.convert("[Internal](.)")
+        self.assertIn("wiki-internal", md_text)
+        self.assertNotIn("wiki-external", md_text)
+        self.assertNotIn("wiki-broken", md_text)
```
Comment (benjaoming) [inline tests/plugins/redlinks/test_redlinks.py]: Could you maybe add something for the parent paths here, too? Would that make sense?

Comment (benjaoming) [reply tests/plugins/redlinks/test_redlinks.py]: I mean like `[Internal](../foo/bar)`

Comment (Mortal) [reply tests/plugins/redlinks/test_redlinks.py]: Certainly - I've pushed a commit with more test coverage, now with tests that use a child article in addition to the root article. Please take another look.

**Comment by benjaoming** [comment]
> Is the file src/wiki/static/wiki/css/all.css actually used by django-wiki anymore? It doesn't seem to be updated by make assets.

Good catch! Also, `print.css` seems to be omitted. I'm not sure what has happened here. Maybe it got lost in the `base_site.html` refactor quite a while back.

https://github.com/django-wiki/django-wiki/blob/master/src/wiki/templates/wiki/base_site.html#L12

Of course, these declarations are easy to add in CSS sources using the `@media` rule.

I think that both `all.css` and `print.css` should be deleted. Looking in `src/wiki/static/wiki/bootstrap/less/wiki/wiki.less`, it seems that the code was moved there and has been updated since.

**Comment by benjaoming** [comment]
Some really nice work there @Mortal - thanks for adding this great new and very useful feature!

**Pull request merged**